### DISCUSSION TITLE
Compiling  test codes successfully ，but the generated executable file does not run.

### DISCUSSION THREAD

**Discussion by yanluo558** [jedisct1/libsodium#1244] [Q&A]
When running the code  /test/default/aead_aes256gcm.c, I compiled successfully ，but the generated executable file does not run.
Here is my code: 
 gcc aead_aes256gcm.c -o a -lsodium
 ./a
Why this happened?

**Comment by jedisct1** [comment]
Hi,

The test suite is compiled and run with the `make check` command.

See the [installation](https://doc.libsodium.org/installation) page in the documentation.
